internal/services/service.go

```diff
@@ -1,38 +1,39 @@
 // (c) Cartesi and individual authors (see AUTHORS)
 // SPDX-License-Identifier: Apache-2.0 (see LICENSE)
 
-// Package services provides mechanisms to start multiple services in the background
+// Package services provides mechanisms to start multiple services in the
+// background
 package services
 
 import (
 	"context"
 	"fmt"
+	"net"
 	"os"
 	"os/exec"
+	"strings"
 	"syscall"
 	"time"
 
 	"github.com/cartesi/rollups-node/internal/logger"
 )
 
-// A service that runs in the background endlessly until the context is canceled
-type Service interface {
-	fmt.Stringer
-
-	// Start a service that will run until completion or until the context is
-	// canceled
-	Start(ctx context.Context) error
-}
-
-const DefaultServiceTimeout = 15 * time.Second
+const (
+	DefaultServiceTimeout  = 15 * time.Second
+	DefaultDialInterval    = 100 * time.Millisecond
+	DefaultStateServerPort = "50051"
+	DefaultHealthcheckPort = "8080"
```

Comment:
I don't like using a default port because it leads to port conflicts.

Reply:
> Can't we make it explicit every time?

Yes, we can. Also, It's better to keep it all together so it becomes easier to throw away when #155 is merged.